setup.py

```diff
@@ -38,7 +36,7 @@
 
 setup(
     name='djangocms-snippet',
-    version=__version__,
+    version='3.0.0',
```

Comment:
This shouldn't need to be changed as it always references the init version which the script also controls / changes.

Reply:
While true, changing it becomes automated, and if someone wanted to check the version and came to `setup.py` instead of `__init__.py` then they'll still see the version rather than having to

Reply:
@Aiky30 Actually I just thought - the options for versions would need extending to account for named & numbered releases for cms 4. However a new version can be specified with;

```
bumpversion --new-version 3.0.1-dev0 patch
```

Do all your cms 4 releases follow the format `{major}.{minor}.{patch}-dev{build}`?

This is likely best done on the v4 support branch to keep the main implementation simple. But I'll do some reading into it.

Reply:
I thought more about the duplicate numbers, it's not very pythonic and people shouldn't need to read the setup.py file, it's parsed by PyPi. Having to be sure that something is updated in 2 places can add issues for maintenance and for anyone cutting a dev release locally. 

For v4 the releases for .dev are done where there's a clear v3 / v4 headache (core cms, snippets, text editor), for any other v4 only they use a standard {major}.{minor}.{patch}.

Reply:
@Aiky30 dont consider them duplicates. I get what you mean, but the whole point of this is that you don't update this number anymore, ever.

I'm working on a consistent and improved developer experience.

Pre commit hooks ensure code quality, then once we're ready for release, bump version handles the version number changes and the changelog using the unreleased items. You just tell bump version if it's a major, minor, or patch release.

Take a look at `setup.cfg` on this PR and you can see what I mean.

Reply:
@marksweb I fully understand what it i and what you're changing. There is always a manual fall back to any automated sequences and for this you have to maintain updating it in 2 places. It's not pythonic to do that. I'm not sure what value it adds to change the setup.py import. It only adds overhead. Automated scripts get changed over time, it's parts like updating a value in multiple places that get broken. I would personally say to keep it DRY.

Reply:
@Aiky30 Ok, thanks - I don't disagree with anything you've said, I just started with the most common/standard implementation. I'll update so that the init holds the version and bumpversion just bumps that.